github/Repository.py

```diff
@@ -788,6 +788,21 @@ def add_to_collaborators(self, collaborator, permission=github.GithubObject.NotS
         return github.Invitation.Invitation(self._requester, headers, data, completed=True) if \
             data is not None else None
 
+    def collaborator_permission(self, collaborator):
```

Comment:
I feel naming it `get_collaborator_permission` is more consistent with other methods, otherwise LGTM

Reply:
@sfdye Done!